tests/python/TestImages.py

```diff
@@ -0,0 +1,357 @@
+        link["ref"] = 32
+        link["direction"] = "in"
+        link["class"] = "Friends"
+
+        addImage = {}
+        addImage["properties"] = props
+        addImage["link"] = link
+
+        imgs_arr = []
+
+        fd = open("../test_images/brain.png")
+        imgs_arr.append(fd.read())
+
+        img_params = {}
+
+        query = {}
+        query["AddImage"] = addImage
+
+        all_queries.append(query)
+
+        # print json.dumps(all_queries)
+        # vdms.aux_print_json(all_queries)
+
+        response, res_arr = db.query(all_queries, [imgs_arr])
+        response = json.loads(response)
+        # vdms.aux_print_json(response)
+
+        self.assertEqual(response[0]["AddEntity"]["status"], 0)
+        self.assertEqual(response[1]["AddImage"]["status"], 0)
+
+    def test_findImage_multiple_res(self):
+        db = vdms.VDMS()
+        db.connect(hostname, port)
+
+        prefix_name = "fimg_brain_multiple"
+
+        number_of_inserts = 4
+        for i in range(0,number_of_inserts):
+            props = {}
+            props["name"] = prefix_name
+            self.insertImage(db, props=props)
+
+        constraints = {}
+        constraints["name"] = ["==", prefix_name]
+
+        results = {}
+        results["list"] = ["name"]
+
+        img_params = {}
+        img_params["constraints"] = constraints
+
+        query = {}
+        query["FindImage"] = img_params
+
+        all_queries = []
+        all_queries.append(query)
+
+        response, img_array = db.query(all_queries)
+        # print vdms.aux_print_json(response)
+
+        response = json.loads(response)
+        self.assertEqual(len(img_array), number_of_inserts)
+        self.assertEqual(response[0]["FindImage"]["status"], 0)
+        self.assertEqual(response[0]["FindImage"]["returned"], number_of_inserts)
+
+    def test_findImageNoBlob(self):
+        db = vdms.VDMS()
+        db.connect(hostname, port)
+
+        prefix_name = "fimg_no_blob_"
+
+        for i in range(0,2):
+            props = {}
+            props["name"] = prefix_name + str(i)
+            self.insertImage(db, props=props)
+
+        all_queries = []
+
+        for i in range(0,2):
+            constraints = {}
+            constraints["name"] = ["==", prefix_name + str(i)]
+
+            results = {}
+            results["return_blob"] = False
+            results["list"] = ["name"]
+
+            img_params = {}
+            img_params["constraints"] = constraints
+            img_params["results"] = results
+
+            query = {}
+            query["FindImage"] = img_params
+
+            all_queries.append(query)
+
+        response, img_array = db.query(all_queries)
+        # print vdms.aux_print_json(response)
+
+        response = json.loads(response)
+        self.assertEqual(response[0]["FindImage"]["status"], 0)
+        self.assertEqual(response[1]["FindImage"]["status"], 0)
+        self.assertEqual(len(img_array), 0)
+
+    # This test is failing.
```

Comment:
Is this test still failing? We might want to take it out for now, until we figure out what's going on

Reply:
Second that